content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-widgets-init.js

```diff
@@ -59,6 +59,29 @@
             $field.prop("checked", $field.attr("value") === value);
         },
 
+        isHiddenField: function ($field) {
+            return !_.isEmpty($field) && ($field.prop("type") === "hidden");
+        },
+
+        setHiddenField: function($field, value) {
+            var $parent = $field.parent();
+            if ($parent.hasClass("coral-DatePicker")) {
+                var displayFormat = $parent.attr("data-displayed-format");
+                var storedFormat = $parent.attr("data-stored-format");
+                var date = moment(new Date(value));
+                var dateFormatted = date.format(displayFormat);
```

Comment:
can you combine all the "var" into single "var" - i am surprised jslint hasnt complained